adoc/headers/id.h

```diff
@@ -33,17 +33,24 @@ template <int Dimensions = 1> class id {
   // OP is: +, -, *, /, %, <<, >>, &, |, ^, &&, ||, <, >, <=, >=
   friend id operatorOP(const id& lhs, const id& rhs) { /* ... */
   }
-  friend id operatorOP(const id& lhs, const size_t& rhs) { /* ... */
+
+  // OP is: +, -, *, /, %, <<, >>, &, |, ^, &&, ||, <, >, <=, >=
+  // Available only when std::is_integral_v<T> is true
+  friend id operatorOP(const id& lhs, const T& rhs) { /* ... */
```

Comment:
Missing the `template<typename T>` in the headers also.

Reply:
Oops.  Fixed in e986d1a2e80cb080f0cd896c5f1cd90eebaade6b.